stanford_jumpstart_engineering.module

```diff
@@ -17,4 +17,18 @@ function stanford_jumpstart_engineering_form_alter(&$form, &$form_state, $form_i
     unset($form['design_container']['styles']['#options']['styles-bright']);
     unset($form['design_container']['styles']['#options']['styles-vivid']);
   }
+
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter() for stanford_news_item_node_form.
+ */
+function stanford_jumpstart_engineering_form_stanford_news_item_node_form_alter(
+  &$form,
+  &$form_state,
+  $form_id) {
+
+  // Handle deprecation of image credits.
+  stanford_news_extras_deprecate_image_credits($form, $form_id);
```

Comment:
plz add sanity check for whether this function exists in case this is pulled onto existing site without most recent news extras.
if function_exists('stanford_news_extras_deprecate_image_credits')....